{
  "cells": [
    {
      "cell_type": "markdown",
      "id": "3e630a32-efb8-452b-a1fa-d28536b8d778",
      "metadata": {
        "id": "3e630a32-efb8-452b-a1fa-d28536b8d778"
      },
      "source": [
        "# RADx Reporter\n",
        "This notebook reports studies submitted by the COVID Rapid Acceleration of Diagnostics [RADx Initiative](https://www.nih.gov/research-training/medical-research-initiatives/radx) to the database of Genotypes and Phenotypes [dbGaP](https://www.ncbi.nlm.nih.gov/gap/) and provides a list of data access requests for the following projects:\n",
        "\n",
        "- [RADx Radical (RADx-rad)](https://www.nih.gov/research-training/medical-research-initiatives/radx/radx-programs#radx-rad)\n",
        "- [RADx Digital Health Technologies (RADx-DHT)](https://www.nih.gov/news-events/news-releases/nih-awards-contracts-develop-innovative-digital-health-technologies-covid-19)\n",
        "- [RADx Tech (RADx-TECH)](https://www.nih.gov/research-training/medical-research-initiatives/radx/radx-programs#radx-tech)\n",
        "- [RADx Underserved Populations (RADx-UP)](https://www.nih.gov/research-training/medical-research-initiatives/radx/radx-programs#radx-up)\n",
        "\n",
        "\n",
        "\n",
        "For a selected RADx project, it retrieves the studies related to the project and the data access requests. Access requests used to test the RADx Data Hub functionality are excluded by default, but can be optionally included.\n",
        "\n",
        "The number of access requests per study is a measure of data reuse. In addition, this notebook queries [Europe PMC](https://europepmc.org/) for publications and preprints that cite or mention dbGaP accession numbers or the RADx project name.\n",
        "\n",
        "Created: 2023-03-09\n",
        "\n",
        "Author : Peter W. Rose (pwrose@ucsd.edu)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "id": "0eb4cad0-4bed-486e-87c6-e2dfe9faeccb",
      "metadata": {
        "id": "0eb4cad0-4bed-486e-87c6-e2dfe9faeccb",
        "outputId": "5bc8373c-c374-4fb4-e940-50f9638af31a",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Project: RADx-rad\n",
            "Exclude test requests: True\n"
          ]
        }
      ],
      "source": [
        "#@title Select a RADx project and then select **Run All** from the **Runtime** menu { run: \"auto\", vertical-output: true, form-width: \"50%\", display-mode: \"form\" }\n",
        "#@markdown ### Select a RADx project\n",
        "project = 'RADx-rad' #@param [\"RADx-rad\",\"RADx-DHT\",\"RADx-TECH\",\"RADx-UP\"]\n",
        "print(f\"Project: {project}\")\n",
        "query = project\n",
        "\n",
        "testers = [\"Rose, Peter \", \"Ciofani, Danielle \", \"Krishnamurthy, Ashok \", \"Claypool, Kajal \", \"Davis-Dusenbery, Brandi \"]\n",
        "#@markdown ### Exclude test requests\n",
        "exclude_tests = True #@param {type:\"boolean\"}\n",
        "print(f\"Exclude test requests: {exclude_tests}\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "id": "ddab1f36-cbd8-4d20-a993-642e5c9467c0",
      "metadata": {
        "tags": [],
        "id": "ddab1f36-cbd8-4d20-a993-642e5c9467c0",
        "cellView": "form"
      },
      "outputs": [],
      "source": [
        "%%capture\n",
        "#@title Installing software on Google Colab\n",
        "#\n",
        "# Install Firefox (see https://github.com/googlecolab/colabtools/issues/3861)\n",
        "#\n",
        "![ ! -f \"/opt/firefox/firefox\" ] && wget 'https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US' -O firefox.tar.bz2 && tar -xjf firefox.tar.bz2 && mv firefox /opt/ && ln -s /opt/firefox/firefox /usr/local/bin/firefox\n",
        "\n",
        "![ ! -f \"/usr/local/bin/geckodriver\" ] && wget 'https://github.com/mozilla/geckodriver/releases/download/v0.33.0/geckodriver-v0.33.0-linux64.tar.gz' && tar -xzf geckodriver*.tar.gz && !chmod +x geckodriver && mv geckodriver /usr/local/bin/\n",
        "\n",
        "![ ! -f \"/usr/local/bin/geckodriver\" ] && apt-get install libxtst6 libdbus-glib-1-2\n",
        "\n",
        "![ ! -f \"/usr/local/bin/geckodriver\" ] && pip install selenium webdriver_manager"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "id": "2991fd05-389b-4203-ba4b-1617ba4d7666",
      "metadata": {
        "tags": [],
        "id": "2991fd05-389b-4203-ba4b-1617ba4d7666",
        "cellView": "form"
      },
      "outputs": [],
      "source": [
        "#@title Importing packages\n",
        "import os\n",
        "import shutil\n",
        "import glob\n",
        "import time\n",
        "from tqdm import tqdm\n",
        "import pandas as pd\n",
        "from urllib.request import urlopen\n",
        "import json\n",
        "from selenium import webdriver\n",
        "from webdriver_manager.firefox import GeckoDriverManager\n",
        "from selenium.webdriver.firefox.service import Service\n",
        "from selenium.webdriver.firefox.options import Options\n",
        "from selenium.webdriver.common.by import By\n",
        "from google.colab import data_table\n",
        "\n",
        "# Add firefox-related .so files to library path\n",
        "os.environ['LD_LIBRARY_PATH'] += ':/opt/firefox'"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "id": "131d482b-6ab3-411c-b7e0-fc42d42a4ec2",
      "metadata": {
        "tags": [],
        "cellView": "form",
        "id": "131d482b-6ab3-411c-b7e0-fc42d42a4ec2",
        "outputId": "643196c4-a481-4948-981f-81993ba39ee5",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "100%|██████████| 15/15 [00:15<00:00,  1.00s/it]\n"
          ]
        }
      ],
      "source": [
        "#@title Running query\n",
        "TMP_DIR = \"/tmp\"\n",
        "filepath = os.path.join(TMP_DIR, \"studies.csv\")\n",
        "\n",
        "def driversetup(download_dir):\n",
        "    options = Options()\n",
        "    options.binary_location = '/usr/local/bin/firefox'\n",
        "\n",
        "    # run Selenium in headless mode\n",
        "    options.add_argument(\"--headless\")\n",
        "    options.add_argument(\"--no-sandbox\")\n",
        "\n",
        "    # set preference to download file to a specified directory\n",
        "    # https://stackoverflow.com/questions/60170311/how-to-switch-download-directory-using-selenium-firefox-python\n",
        "    # 0: download to the desktop, 1 download to the default \"Downloads\" directory, 2 use specified directory\n",
        "    options.set_preference(\"browser.download.folderList\", 2)\n",
        "    options.set_preference(\"browser.download.manager.showWhenStarting\", False)\n",
        "    options.set_preference(\"browser.download.dir\", download_dir)\n",
        "    options.set_preference(\"browser.helperApps.neverAsk.saveToDisk\", \"text/csv\")\n",
        "\n",
        "    # instantiate driver\n",
        "    service = Service(executable_path=GeckoDriverManager().install(), log_output=\"geckodriver.log\")\n",
        "    driver = webdriver.Firefox(service=service, options=options)\n",
        "    driver.implicitly_wait(5)\n",
        "\n",
        "    return driver\n",
        "\n",
        "def download_dbgap_studies(query, filepath):\n",
        "    # clean up any previously downloaded csv files\n",
        "    files = glob.glob(os.path.join(TMP_DIR, \"*.csv\"))\n",
        "    for file in files:\n",
        "        os.remove(file)\n",
        "\n",
        "    # run advanced search\n",
        "    driver = driversetup(TMP_DIR)\n",
        "    driver.get(f\"https://www.ncbi.nlm.nih.gov/gap/advanced_search/?TERM={query}\")\n",
        "    time.sleep(3)\n",
        "\n",
        "    # download csv file\n",
        "    button = driver.find_element(By.CLASS_NAME, \"usa-button-secondary\")\n",
        "    time.sleep(3)\n",
        "    button.click()\n",
        "\n",
        "    # wait until download is completed\n",
        "    for step in tqdm(range(15)):\n",
        "        time.sleep(1)\n",
        "\n",
        "    driver.close()\n",
        "\n",
        "    # move downloaded csv file to a standard location\n",
        "    move_studies_file(filepath)\n",
        "\n",
        "def move_studies_file(filepath):\n",
        "    \"\"\" Move downloaded file to a specified standard location\"\"\"\n",
        "    # the file name of the downloaded csv file is unknown in advance,\n",
        "    # but there should be only one csv file.\n",
        "    files = glob.glob(os.path.join(TMP_DIR, \"*.csv\"))\n",
        "    if len(files) == 1:\n",
        "        shutil.move(files[0], filepath)\n",
        "    else:\n",
        "        print(\"query error\")\n",
        "\n",
        "filepath = \"studies.csv\"\n",
        "download_dbgap_studies(query, filepath)\n",
        "\n",
        "studies = pd.read_csv(filepath, usecols=[\"accession\", \"name\", \"description\", \"Study Design\", \"Study Consent\",])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "id": "82a725e0-c840-4854-a927-69b07836258b",
      "metadata": {
        "tags": [],
        "cellView": "form",
        "id": "82a725e0-c840-4854-a927-69b07836258b",
        "outputId": "7daec1be-ab96-40cd-863d-82bc2421944e",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 744
        }
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Number of studies for RADx-rad: 48\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>accession</th>\n",
              "      <th>name</th>\n",
              "      <th>description</th>\n",
              "      <th>Study Design</th>\n",
              "      <th>Study Consent</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>phs002964.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Self-report of sudden loss of smell or taste s...</td>\n",
              "      <td>Prospective Longitudinal Cohort</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>phs002945.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The primary objective of this study was to des...</td>\n",
              "      <td>Cross-Sectional</td>\n",
              "      <td>GRU --- General Research Use, GRU-IRB --- Gene...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>phs002924.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>This project will develop biomimetic olfaction...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>phs002782.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Infectious disease outbreaks like Coronavirus ...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>phs002781.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>SARS-CoV-2 infection exhibits a wide range of ...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>phs002778.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Robust, efficient, and reliable testing for SA...</td>\n",
              "      <td>Case-Control</td>\n",
              "      <td>HMB --- Health/Medical/Biomedical</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>phs002747.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Recent studies, including ours, have suggested...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>phs002744.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The ultimate goal of this proposal is to devel...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>phs002729.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Presently, the application of molecular techno...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>phs002709.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>COVID-19, a global pandemic due to its rapid p...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td>phs002702.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>This study is designed to develop an aptamer-b...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11</th>\n",
              "      <td>phs002700.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Coronavirus 2019 (COVID-19) has afflicted 6.2 ...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>phs002699.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Smell loss is a predominant symptom of COVID-1...</td>\n",
              "      <td>Cross-Sectional</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>phs002689.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The pandemic caused by COVID-19 has been assoc...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>14</th>\n",
              "      <td>phs002685.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Automated, rapid diagnostics with little sampl...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>15</th>\n",
              "      <td>phs002679.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>When faced with a pandemic such as SARS-Corona...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>16</th>\n",
              "      <td>phs002657.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>With older age and multiple comorbidities, dia...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>17</th>\n",
              "      <td>phs002642.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The goals of this Emergency Competitive Revisi...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>18</th>\n",
              "      <td>phs002631.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The COVID-19 pandemic is the most devastating ...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>19</th>\n",
              "      <td>phs002609.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The purpose of this study is to develop a tool...</td>\n",
              "      <td>Case-Control</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>20</th>\n",
              "      <td>phs002604.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Wastewater based testing (WBT) holds great pro...</td>\n",
              "      <td>Prospective Longitudinal Cohort</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>21</th>\n",
              "      <td>phs002603.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The recent emergence of SARS-CoV2 and resultan...</td>\n",
              "      <td>Prospective Longitudinal Cohort</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>22</th>\n",
              "      <td>phs002602.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>To help diversify the approaches that can be u...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>23</th>\n",
              "      <td>phs002600.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The data herein combines GC-MS and GC-DMS anal...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>24</th>\n",
              "      <td>phs002588.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>COVID-19 presents a public health emergency. T...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>25</th>\n",
              "      <td>phs002585.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>This work is directed at characterizing pediat...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>26</th>\n",
              "      <td>phs002583.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>We propose to develop a novel testing platform...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>27</th>\n",
              "      <td>phs002573.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The goal of the project is to develop a pipeli...</td>\n",
              "      <td>Case-Control</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>28</th>\n",
              "      <td>phs002572.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>A substantial and growing body of evidence has...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>29</th>\n",
              "      <td>phs002570.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>This project will develop a multiplexed sensor...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>30</th>\n",
              "      <td>phs002569.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>This study's objective is to understand which ...</td>\n",
              "      <td>Case-Control</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>31</th>\n",
              "      <td>phs002565.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>We are developing an electrochemical assay for...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>32</th>\n",
              "      <td>phs002563.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The COVID-19 pandemic has caused unprecedented...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>33</th>\n",
              "      <td>phs002561.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>A rapid, sensitive, low-cost diagnostic device...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>34</th>\n",
              "      <td>phs002553.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>As the COVID-19 pandemic evolved in early 2020...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>35</th>\n",
              "      <td>phs002551.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Marshallese Pacific Islanders bear a dispropor...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>36</th>\n",
              "      <td>phs002550.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Nucleic acid test-standard for diagnostics tes...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>37</th>\n",
              "      <td>phs002549.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Children have been disproportionately less imp...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>38</th>\n",
              "      <td>phs002546.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The devastation caused by emerging pathogens w...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>39</th>\n",
              "      <td>phs002544.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>This U18 application is responsive to the NIH'...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>40</th>\n",
              "      <td>phs002543.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Reliable real-time detection of SARS-CoV-2 vir...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>41</th>\n",
              "      <td>phs002542.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Developing next generation technology to simpl...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>42</th>\n",
              "      <td>phs002527.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>COVID-19 is expected to become one of the larg...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>43</th>\n",
              "      <td>phs002525.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The University of Miami (UM), with three prima...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>44</th>\n",
              "      <td>phs002524.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>Vulnerable populations do not just need testin...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>45</th>\n",
              "      <td>phs002523.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>We plan to extend and improve our COH/PHD syst...</td>\n",
              "      <td>Collection</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>46</th>\n",
              "      <td>phs002522.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>This proposal describes the design, fabricatio...</td>\n",
              "      <td>Methods</td>\n",
              "      <td>GRU --- General Research Use</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>47</th>\n",
              "      <td>phs003124.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "      <td>The urgent need to curb the spread of SARS-CoV...</td>\n",
              "      <td>Case Set</td>\n",
              "      <td>HMB --- Health/Medical/Biomedical</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/881c4a0d49046431/data_table.js\";\n\n      const table = window.createDataTable({\n        data: [[\"phs002964.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Longitudinal at    Home Smell Testing to Detect Infection by SARS-CoV-2\",\n\"Self-report of sudden loss of smell or taste substantially increases the odds of being diagnosed with SARS-CoV-2. However, self-report of smell function is an unreliable predictor of smell loss. We   work, and instead seek an evaluation by a healthcare professional. Note for data in RADx: Instructions for requesting individual-level data are available\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General Research Use\"],\n [\"phs002945.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Data Science    Approach to Identify and Manage Multisystem Inflammatory Syndrome in    Children (MIS-C) Associated with SARS-CoV-2 Infection and Kawasaki    Disease in Pediatric Patients\",\n\"The primary objective of this study was to design and validate a predictive decision support system for the identification, treatment, and management of SARS-CoV-2 associated with multisystem inflammatory syndrome in   part of this study. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov\",\n\"Cross-Sectional\",\n\"GRU --- General Research Use, GRU-IRB --- General Research Use (IRB)\"],\n [\"phs002924.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Broad-Spectrum    Detection of VOC and Non-VOC Biomarkers from Patient Exhalant Using a    Biomimetic Multiplexed eNose Biosensor for COVID 19 Diagnosis\",\n\"This project will develop biomimetic olfaction-based and gustation-based sensors for detecting volatile organic compound (VOC) biomarkers against SARS-CoV-2. We employ a computational design to deduce sets of odorant binding peptide   test for SARS-CoV-2 based on direct exhaled breath. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002782.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): NIEHS    Diagnostic-Prognostic RNAseq\",\n\"Infectious disease outbreaks like Coronavirus Disease 2019 (COVID-19) can overwhelm healthcare systems when screening tools are scarce or lacking. In the face of an ongoing COVID-19 pandemic and with single-plex  , long queue times, backlogs in COVID-19 diagnoses, and delayed access to specialized treatment for COVID-19 patients. The goal of this RADx funded project\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002781.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Discovery and    Clinical Validation of Host Biomarkers of Disease Severity and    Multi-system Inflammatory Syndrome in Children (MIS-C) and COVID-19\",\n\"SARS-CoV-2 infection exhibits a wide range of clinical outcomes in both children and adults, from asymptomatic and mild disease to severe viral pneumonia, ARDS, acute kidney injury, thrombotic disorders, and   (EUA). Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002778.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Microfluidic    Isolation and Characterization of SARS-CoV-2 and Virus Related    Exosomes\",\n\"Robust, efficient, and reliable testing for SARS-CoV-2 is extraordinarily challenging due to our lack of ultra-sensitive assays and constantly evolving knowledge of the virus. Standard PCR based assays still result   authorization of our test to be used in the clinic. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at\",\n\"Case-Control\",\n\"HMB --- Health/Medical/Biomedical\"],\n [\"phs002747.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): COVID-19    Detection through Scent Analysis with a Compact Gas Chromatography    Device\",\n\"Recent studies, including ours, have suggested that a person's breath may used to diagnose COVID-19 infection and even to monitor its progress. In contrast to immunological and genetic based methods   analyses on up to 760 patients, and to identify and validate COVID-19 biomarkers in breath. Thus, in coordination with the RADx-rad Data Coordination Center\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002744.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Rolosense: An    Innovative Platform for Automatic Mobile Phone Readout of Active    SARS-CoV-2\",\n\"The ultimate goal of this proposal is to develop a novel platform technology for automatic surveillance and tracing of airborne SARS-CoV-2 virus particles in real time. The centerpiece of this   tracing needs. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002729.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Improved    Scalability, Sensitivity, and Interpretability of Pathogen Detection,    Including SARS-CoV-2, in Wastewater using High-Throughput, Highly    Multiplexed Digital Array PCR Technology\",\n\"Presently, the application of molecular technologies, such as, RT-qPCR and digital PCR (dPCR) to quantify SARS-CoV-2 and related targets in wastewater is cumbersome, time consuming, and costly. While progress has   pathogen threats in the future. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002709.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multiparametric    Integrated Molecular Detection of SARS-CoV-2 from Biofluids by    Adapting Single Extracellular Vesicle Characterization Technologies\",\n\"COVID-19, a global pandemic due to its rapid progression, is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). There are different assays for COVID-19, including nucleic acid, serological,   nasopharyngeal secretions and saliva from COVID-19 patients. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002702.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Scalable    Aptamer-Based Electrochemical Biosensor for Rapid Detection of    SARS-CoV-2 from Saliva\",\n\"This study is designed to develop an aptamer-based electrochemical detection platform with SARS-CoV-2 as the model system to demonstrate its performance. As this study is a technology development, the data   prototype platform developed as a result of this technology-based development project supported by the NIH. Note for data in RADx: Instructions for requesting\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002700.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Nanobody-based    Electrochemical Biosensor for Real-Time Detection of Aerosolized    SARS-CoV2\",\n\"Coronavirus 2019 (COVID-19) has afflicted 6.2 million Americans and killed 190,000 as of early September 2020 (WHO website); a roughly 3% mortality. Between a shortage in testing and unidentified asymptomatic   future pathogens, would enable us to limit the viral spread throughout the community in the current and future pandemics. Note for data in RADx\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002699.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SCENTinel: A    Rapid Smell Test for COVID-19 Surveillance\",\n\"Smell loss is a predominant symptom of COVID-19, and initial evidence based on self-reports suggests that chemosensory loss is a sensitive predictor of COVID-19 in the general population, more so   relation to a confirmed COVID-19 diagnosis, providing a crucially needed means to contain the spread of COVID-19. Note for data in RADx: Instructions for\",\n\"Cross-Sectional\",\n\"GRU --- General Research Use\"],\n [\"phs002689.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid Saliva    Antigen Test for SARs-CoV-2 Detection\",\n\"The pandemic caused by COVID-19 has been associated with a very high disease burden as well as high global financial costs. With this pandemic came massive shut downs of both   proteins and developed the SARS-CoV-2 assays. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002685.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): DNA Star    SAS-CoV-2 Rapid Test\",\n\"Automated, rapid diagnostics with little sample collection and preparation are needed to identify and trace affected persons in times when hyper-infectious pathogens cause pandemics. Frequent, low cost and highly scalable   to results in minutes) and cost effective ( $3 per test). Note for data in RADx: Instructions for requesting individual-level data are available on\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002679.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Wastewater    Detection of COVID-19\",\n\"When faced with a pandemic such as SARS-Coronavirus-2 (SAR-CoV-2), the virus responsible for COVID-19, timely risk assessment and action are required to prevent public health impacts to entire communities. Because   existing and emerging variants from wastewater, and 3) design platforms for communicating wastewater variant results to the public. Note for data in RADx\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002657.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Early Detection,    Containment, and Management of COVID-19 in Dialysis Facilities Using    Multi-Modal Data Sources\",\n\"With older age and multiple comorbidities, dialysis patients are at high risk for serious complications, even death, from COVID-19. There is a large disproportionate representation of minorities, especially Blacks and   the high-risk population. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002642.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A    Confectionary-Based Screening Tool for Assessing Chemosensory Loss in    COVID-19 Patients\",\n\"The goals of this Emergency Competitive Revision are to develop (Aim 1) and deploy (Aim 2) a novel, objective, psychophysical smell and taste screening test to detect the onset of   site to perform this urgent research. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002631.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Rapid Olfactory    Tools for Telemedicine-Friendly COVID-19 Screening and Survey\",\n\"The COVID-19 pandemic is the most devastating infectious disease outbreak in a century, particularly in underserved and minoritized communities. In 2020 alone, it costed a million lives. It continues to   been traditionally underserved by the health care system and public health infrastructure. Note for data in RADx: Instructions for requesting individual\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002609.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Identifying    Biomarker Signatures of Prognostic Value for Multisystem Inflammatory    Syndrome in Children (MIS-C)\",\n\"The purpose of this study is to develop a tool that can diagnose multisystem inflammatory syndrome in children (MIS-C.) Some children and young adults who have had Covid-19 develop this   others do not. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home\",\n\"Case-Control\",\n\"GRU --- General Research Use\"],\n [\"phs002604.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Tracking the    COVID-19 Epidemic in Sewage (TRACES)\",\n\"Wastewater based testing (WBT) holds great promise for cost-effective population surveillance and transmission tracking of SARS-CoV-2, but optimal sampling modalities and protocols are unknown. Taking advantage of a diverse inner   develop point-of-use microfluidics systems for timely WBT. Note for data in RADx: Instructions for requesting individual-level data are available on RADx\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General Research Use\"],\n [\"phs002603.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Diagnosis of    MIS-C in Febrile Children\",\n\"The recent emergence of SARS-CoV2 and resultant pandemic of COVID-19 disease has overwhelmed global health systems and led to over 200,000 American deaths to date. While initial reports suggested that   diagnostic strategy to distinguish children with MIS-C from children with other causes of fever. Note for data in RADx: Instructions for requesting individual\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General Research Use\"],\n [\"phs002602.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Virus Counter:    Rapid and Sensitive Diagnostics Based on Digital Detection of    Individual Pathogens\",\n\"To help diversify the approaches that can be used to sensitively detect SARS-CoV-2 and improve response resilience, we will pivot an established virus detection system that has already shown the   instrument. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002600.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Portable GC    Detector for COVID Diagnostics\",\n\"The data herein combines GC-MS and GC-DMS analysis of exhaled breath vapor compounds. The intent of this study is to develop a portable GC-DMS system to diagnose SARS-CoV-2 infections from  , weight, symptoms at time of sampling, etc., was also collected. Note for data in RADx: Instructions for requesting individual-level data are available on\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002588.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Effective,    Reagent-Free Detection of the Odor Signature of COVID-19 Infection    Using a Nano-Enabled Sensor Array\",\n\"COVID-19 presents a public health emergency. There is a critical need for rapid, not reagent intensive, non-invasive testing technologies. This program will lead to the production of a prototype system   clear the proposed COVID-19 testing device as Class I or Class II without reference to any predicate. Note for data in RADx: Instructions for requesting\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002585.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): AICORE-kids\",\n\"This work is directed at characterizing pediatric COVID-19 and stratifying incoming patients by projected (future) disease severity. Such stratification has several implications: immediately improving treatment planning, and as disease mechanistic   regulatory milestones intended to conform with the Emergency Use Authorization (EUA) programs in effect for SARS-CoV-2 diagnostics. Note for data in RADx\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002583.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid    Breathalyzer Diagnostics Platform for COVID-19\",\n\"We propose to develop a novel testing platform that detects SARS-CoV-2 virions in a patient's breath. When a person exhales into the COVID breathalyzer, droplets and other emitted particles are  , and has accurate reporting with high sensitivity and specificity. Note for data in RADx: Instructions for requesting individual-level data are available\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002573.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Detection and    Automatic Privacy-Protected Contact Tracing System Designed for    COVID-19\",\n\"The goal of the project is to develop a pipeline for automatic detection and tracing of active SARS-CoV-2 infection in a cost-effective, highly accurate, real-time, portable, and super-encrypted manner. Our   transition and tracing system. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx\",\n\"Case-Control\",\n\"GRU --- General Research Use\"],\n [\"phs002572.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A SARS-CoV-2    Breathalyzer for Direct Virus Detection\",\n\"A substantial and growing body of evidence has demonstrated that COVID-19 is transmitted by human-emitted airborne particles; therefore, it is critical to rapidly screen individuals to determine whether they are   bring the current COVID-19 pandemic under control before this SARS-CoV-2 breathalyzer is commercially available. Note for data in RADx: Instructions for\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002570.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Multimodal    Platform for Oral Screening of COVID-19\",\n\"This project will develop a multiplexed sensor platform based on nanomaterials capable of rapid simultaneous detection of multiple components of viral particles in a field-applicable electrochemical device. The device has   data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for data access in\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002569.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): COVID-19 Network    of Networks Expanding Clinical and Translational Approaches to Predict    Severe Illness in Children (CONNECT to Predict SIck Children)\",\n\"This study's objective is to understand which children are at the highest risk for experiencing severe consequences from SARS-CoV-2 infection. The study will use clinical, epidemiologic, and sociodemographic data, alongside   for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for data\",\n\"Case-Control\",\n\"GRU --- General Research Use\"],\n [\"phs002565.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad):    Touchscreen-Compatible, Real-Time Electrochemical Sensing of    SARS-CoV-2\",\n\"We are developing an electrochemical assay for detecting SARS-CoV-2. We will be generating data that includes a signal for a given virus or spike (S) protein concentration, as well as  binding constants for molecular recognition agents. Note for data in RADx: Instructions for requesting individual-level data are available on RADx\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002563.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Handheld    Microchip for GC Analysis of Breath to Screen for COVID-19\",\n\"The COVID-19 pandemic has caused unprecedented societal suffering and economic disruption. Although current COVID-19 diagnostic testing technologies are critical for curbing the spread of the virus and preventing future outbreaks,   REACH program. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002561.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid,    Sensitive, Point-of-Care, Antigen-Based Diagnostics for SARS-CoV-2\",\n\"A rapid, sensitive, low-cost diagnostic device for early detection of novel Coronavirus Disease (COVID-19) could significantly change the paradigm in real-time and efficient management of patients and available healthcare resources,   cartridge. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002553.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Characterization    of Multisystem Inflammatory Syndrome in Children and its Relationship    to Kawasaki Disease\",\n\"As the COVID-19 pandemic evolved in early 2020, case reports appeared describing children with unusual febrile illnesses with elevated inflammatory markers and multi-system involvement that is now termed Multisystem Inflammatory  -of-service RNA or antibody profiling with commercial partners. De-identified clinical and molecular data will be deposited in the RADx-rad hub to\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002551.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Marshallese:    Alternate Surveillance for COVID-19 in a Unique Population\",\n\"Marshallese Pacific Islanders bear a disproportionate burden of COVID-19 infection, hospitalization, and death, with rates 4 to 25 times higher than those of other US racial and ethnic groups in   populations. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002550.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Direct    Bioelectronic Detection of SARS-CoV-2 From Saliva Using Singlemolecule    Field-effect Transistor Array\",\n\"Nucleic acid test-standard for diagnostics testing for COVID-19, usually performed in specialized laboratories. Most are based on reverse-transcription quantitative polymerase chain reaction (qRT-PCR). The time required for specimen transport and   diagnosis applications. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002549.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Severity    Predictors Using Immunology and Transcriptomics in Saliva Using Multi    Neural Network Intelligence in SARS-CoV2 Infection in Children (SPITS    MISC)\",\n\"Children have been disproportionately less impacted by the Coronavirus Disease 2019 (COVID-19) caused by the Severe Acute Respiratory Syndrome Coronavirus 2 (SAR-CoV-2) compared to adults. However, severe illnesses including Multisystem   appropriate resource allocation during the pandemic. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002546.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Development of an    Automated Diagnostic Platform for SARS-CoV-2 Monitoring in Vulnerable    Areas\",\n\"The devastation caused by emerging pathogens with fast transmission capacity, such as SARS-CoV-2, has demonstrated the importance of preparedness for future viral outbreaks; this includes the ability for fast deployment   may repeat over several decades, much like the occasional visits from the influenza virus, with varying degrees of virulence. Note for data in RADx\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002544.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): AFS/SERS    Saliva-Based SARS-CoV-2 Earliest Infection and Antibodies Detection\",\n\"This U18 application is responsive to the NIH's RADx-rad Emergency Response to the COVID-pandemic for new or non-traditional technologies developed to single extracellular vesicle, exosome and extracellular RNA (exRNA) isolation   patients. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002543.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Minimal    False-Alarm Touch-Based Detection of SARS-CoV-2 Virus Particles Using    Poly-Aptamers\",\n\"Reliable real-time detection of SARS-CoV-2 virus represents a considerable challenge, with available tools requiring extensive sample preparation and/or expensive lab-based equipment to obtain accurate results. The objective in this proposal   innovations and the prior scientific and engineering accomplishments of the team. Note for data in RADx: Instructions for requesting individual-level data are\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs002542.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Wastewater    Assessment for Coronavirus in Kentucky: Implementing Enhanced    Surveillance Technology\",\n\"Developing next generation technology to simplify wastewater RNA extraction and quantification will make wastewater surveillance feasible and scalable in facilities and rural communities. Wastewater testing for SARS-CoV-2 in rural communities  , and utility of wastewater surveillance for SARS-CoV-2 and identify community measures taken in response to test results. Note for data in RADx\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002527.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Bioinformatics    Framework for Wastewater-based Surveillance of Infectious Diseases\",\n\"COVID-19 is expected to become one of the largest mass casualty events in the history of the United States (U.S.). Assessment of the true burden of disease in the population   and how these burdens differ across communities. The goal of our proposed RADx-rad supplement is to implement and evaluate a near real-time WBE\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002525.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SF-RAD:    Development and Proof-of-Concept Implementation of the South Florida    Miami RADx-rad SARS-CoV-2 Wastewater-Based Surveillance Infrastructure\",\n\"The University of Miami (UM), with three primary campuses in Miami, Florida, is geographically spread within one of the worst current COVID-19 hotbeds. UM has deployed an elaborate human surveillance   strategies. Working closely with the RADx-rad Data Coordination Center (DCC), this application (SF-RAD) will develop and implement data standards and\",\n\"Case Set\",\n\"GRU --- General Research Use\"],\n [\"phs002524.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Validation of    Smart Masks for Surveillance of COVID-19\",\n\"Vulnerable populations do not just need testing - they need surveillance. The ideal surveillance tool would operate in the background with minimal involvement of the population to be tested; it  . Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for data\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002523.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multi-Modal    Wireless COVID Monitoring and Infection Alerts for Concentrated    Populations\",\n\"We plan to extend and improve our COH/PHD system in using wearables for the early detection of COVID-19 infection. With the over 5,000 patients who have been recruited in our  ). Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for data\",\n\"Collection\",\n\"GRU --- General Research Use\"],\n [\"phs002522.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): MOF-SCENT:    Metal-Organic Frameworks for Screening COVID-19 by Electronic-Nose    Technology to Improve Selectivity and Time Response\",\n\"This proposal describes the design, fabrication, and proof-of-concept testing of a novel MOF-SCENT system (Metal-Organic Frameworks for Screening COVID-19 by Electronic-Nose Technology) that features the following advantages: (1) Highly selective   in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home. Apply for data access in\",\n\"Methods\",\n\"GRU --- General Research Use\"],\n [\"phs003124.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Exosome-Based    Non-Traditional Technologies Towards Multi-Parametric and Integrated    Approaches for SARS-CoV-2\",\n\"The urgent need to curb the spread of SARS-CoV-2 demands availability of diagnostics that are more rapid, accurate, sensitive and affordable than qPCR and antibody tests. Current qPCR tests are   testing of workers. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data Hub at https://radx-hub.nih.gov/home\",\n\"Case Set\",\n\"HMB --- Health/Medical/Biomedical\"]],\n        columns: [[\"string\", \"accession\"], [\"string\", \"name\"], [\"string\", \"description\"], [\"string\", \"Study Design\"], [\"string\", \"Study Consent\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n      \n      function appendQuickchartButton(parentElement) {\n        let quickchartButtonContainerElement = document.createElement('div');\n        quickchartButtonContainerElement.innerHTML = `\n    <div id=\"df-8a27f809-30c7-4f88-bb31-7e20ffa123aa\">\n      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-8a27f809-30c7-4f88-bb31-7e20ffa123aa')\"\n              title=\"Suggest charts.\"\n              style=\"display:none;\">\n\n<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n     width=\"24px\">\n    <g>\n        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n    </g>\n</svg>\n      </button>\n    </div>\n\n<style>\n  .colab-df-quickchart {\n    background-color: #E8F0FE;\n    border: none;\n    border-radius: 50%;\n    cursor: pointer;\n    display: none;\n    fill: #1967D2;\n    height: 32px;\n    padding: 0 0 0 0;\n    width: 32px;\n  }\n\n  .colab-df-quickchart:hover {\n    background-color: #E2EBFA;\n    box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n    fill: #174EA6;\n  }\n\n  [theme=dark] .colab-df-quickchart {\n    background-color: #3B4455;\n    fill: #D2E3FC;\n  }\n\n  [theme=dark] .colab-df-quickchart:hover {\n    background-color: #434B5C;\n    box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n    filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n    fill: #FFFFFF;\n  }\n</style>\n\n    <script>\n      async function quickchart(key) {\n        const containerElement = document.querySelector('#' + key);\n        const charts = await google.colab.kernel.invokeFunction(\n            'suggestCharts', [key], {});\n      }\n    </script>\n`;\n        parentElement.appendChild(quickchartButtonContainerElement);\n        \nfunction displayQuickchartButton(domScope) {\n  let quickchartButtonEl =\n    domScope.querySelector('#df-8a27f809-30c7-4f88-bb31-7e20ffa123aa button.colab-df-quickchart');\n  quickchartButtonEl.style.display =\n    google.colab.kernel.accessAllowed ? 'block' : 'none';\n}\n\n        displayQuickchartButton(parentElement);\n      }\n\n      appendQuickchartButton(table);\n    ",
            "text/plain": [
              "<google.colab.data_table.DataTable object>"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "#@title Table of studies\n",
        "print(f\"Number of studies for {project}:\", studies.shape[0])\n",
        "if studies.shape[0] > 0:\n",
        "    display(data_table.DataTable(studies, include_index=False, num_rows_per_page=10))"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Summary of data access requests\n",
        "def get_download_url(accession):\n",
        "    return \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetAuthorizedRequestDownload.cgi?study_id=\" + accession\n",
        "\n",
        "def get_authorized_requests(studies):\n",
        "    authorized_requests = pd.DataFrame()\n",
        "\n",
        "    for _, row in tqdm(studies.iterrows(), total=studies.shape[0]):\n",
        "        try:\n",
        "            df = pd.read_csv(get_download_url(row[\"accession\"]),\n",
        "                             usecols=[\"Requestor\", \"Affiliation\", \"Project\", \"Date of approval\", \"Request status\",\n",
        "                                      \"Public Research Use Statement\", \"Technical Research Use Statement\"],\n",
        "                            sep=\"\\t\")\n",
        "            df[\"accession\"] = row[\"accession\"]\n",
        "            df[\"name\"] = row[\"name\"]\n",
        "            authorized_requests = pd.concat([authorized_requests, df], ignore_index=True)\n",
        "        except:\n",
        "            print(f\"Skipping: {row['accession']} - no data access through dbGaP.\")\n",
        "\n",
        "    return authorized_requests\n",
        "\n",
        "requests = get_authorized_requests(studies)\n",
        "\n",
        "# exclude test requests\n",
        "n_excluded = 0\n",
        "if exclude_tests:\n",
        "    n_excluded = requests.shape[0]\n",
        "    requests = requests[~requests[\"Requestor\"].isin(testers)]\n",
        "    n_excluded = n_excluded - requests.shape[0]\n",
        "\n",
        "\n",
        "# group requests to create a summary view\n",
        "if requests.shape[0] > 0:\n",
        "    summary = requests.groupby([\"Requestor\", \"Affiliation\", \"Project\", \"Date of approval\", \"Request status\",\n",
        "                                \"Public Research Use Statement\", \"Technical Research Use Statement\"],\n",
        "                                as_index=False)[\"accession\"].agg(', '.join)\n",
        "\n",
        "    summary[\"Number of requests\"] = summary[\"accession\"].str.count(\",\") + 1\n",
        "\n",
        "# show results\n",
        "print()\n",
        "print()\n",
        "if exclude_tests:\n",
        "        print(f\"{n_excluded} test requests have been excluded!\")\n",
        "\n",
        "print(\"Number of data access requests :\", requests.shape[0])\n",
        "\n",
        "if requests.shape[0] > 0:\n",
        "    print(\"Number of unique requestors    :\", len(requests[\"Requestor\"].unique()))\n",
        "    print(\"Number of unique studies       :\", len(requests[\"accession\"].unique()))\n",
        "    display(data_table.DataTable(summary, include_index=False, num_rows_per_page=10))"
      ],
      "metadata": {
        "cellView": "form",
        "id": "Hf23L_FrJPiJ",
        "outputId": "25604c15-425b-42d2-967e-b64a909d8a30",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 688
        }
      },
      "id": "Hf23L_FrJPiJ",
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "100%|██████████| 48/48 [00:36<00:00,  1.32it/s]"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "49 test requests have been excluded!\n",
            "Number of data access requests : 4\n",
            "Number of unique requestors    : 3\n",
            "Number of unique studies       : 4\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Requestor</th>\n",
              "      <th>Affiliation</th>\n",
              "      <th>Project</th>\n",
              "      <th>Date of approval</th>\n",
              "      <th>Request status</th>\n",
              "      <th>Public Research Use Statement</th>\n",
              "      <th>Technical Research Use Statement</th>\n",
              "      <th>accession</th>\n",
              "      <th>Number of requests</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Anwar, Mohd Mozharul</td>\n",
              "      <td>NIH</td>\n",
              "      <td>Exploration of Wearable Device Data for COVID-19</td>\n",
              "      <td>Mar27, 2023</td>\n",
              "      <td>approved</td>\n",
              "      <td>Wearable devices collect various physiological...</td>\n",
              "      <td>The objective of the proposed research is to e...</td>\n",
              "      <td>phs002523.v1.p1</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>Miguez, Maria-Jose</td>\n",
              "      <td>FLORIDA INTERNATIONAL UNIVERSITY</td>\n",
              "      <td>System analysis for COVID humoral response</td>\n",
              "      <td>Feb10, 2023</td>\n",
              "      <td>approved</td>\n",
              "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
              "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
              "      <td>phs002945.v1.p1, phs002781.v1.p1</td>\n",
              "      <td>2</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>Solo-Gabriele, Helena</td>\n",
              "      <td>UNIVERSITY OF MIAMI CORAL GABLES</td>\n",
              "      <td>Request to access SF-RAD: Development of Proof...</td>\n",
              "      <td>Jul26, 2023</td>\n",
              "      <td>approved</td>\n",
              "      <td>I am the Principal Investigator for the projec...</td>\n",
              "      <td>I am the Principal Investigator for the projec...</td>\n",
              "      <td>phs002525.v1.p1</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/881c4a0d49046431/data_table.js\";\n\n      const table = window.createDataTable({\n        data: [[\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Mar27, 2023\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002523.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Miguez, Maria-Jose \",\n\"FLORIDA INTERNATIONAL UNIVERSITY\",\n\"System analysis for COVID humoral response\",\n\"Feb10, 2023\",\n\"approved\",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Therefore, it is critical to understand who is at risk (e.g. those with allergies). Our long term goal is to improve early diagnosis.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely prepared given our medical and research experience.   \",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Indeed, in a sizable proportion of cases (up to 40%), the patient requires to be admitted to the ICU service. Therefore, it is critical to understand who is at risk (e.g. those with allergic or non-allergic diseases) those with alterations in the IgE network (platelets, eosinophils, specific Th2 or Th17 cytokines). Our long term objective is to develop new strategies for early diagnosis and to identify biomarkers of risk that can be easily deployed globally.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely qualified to do such analyses given the medical background of the PI, our ongoing work in COVID both in humans and with animal models, our unique expertise in IgE, and expertise using machine learning and model-based analyses.   \",\n\"phs002945.v1.p1, phs002781.v1.p1\",\n{\n            'v': 2,\n            'f': \"2\",\n        }],\n [\"Solo-Gabriele, Helena \",\n\"UNIVERSITY OF MIAMI CORAL GABLES\",\n\"Request to access SF-RAD: Development of Proof-of-Concept\",\n\"Jul26, 2023\",\n\"approved\",\n\"I am the Principal Investigator for the project and I would like to download the data to see how it looks from the user's perspective.  This will help us improve the data upload.\",\n\"I am the Principal Investigator for the project and I would like to download the data to see how it looks from the user's perspective.  This will help us improve the data upload.\",\n\"phs002525.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }]],\n        columns: [[\"string\", \"Requestor\"], [\"string\", \"Affiliation\"], [\"string\", \"Project\"], [\"string\", \"Date of approval\"], [\"string\", \"Request status\"], [\"string\", \"Public Research Use Statement\"], [\"string\", \"Technical Research Use Statement\"], [\"string\", \"accession\"], [\"number\", \"Number of requests\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n      \n      function appendQuickchartButton(parentElement) {\n        let quickchartButtonContainerElement = document.createElement('div');\n        quickchartButtonContainerElement.innerHTML = `\n    <div id=\"df-b4c52e64-21c2-4d5d-bd8a-c20b9852e44b\">\n      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-b4c52e64-21c2-4d5d-bd8a-c20b9852e44b')\"\n              title=\"Suggest charts.\"\n              style=\"display:none;\">\n\n<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n     width=\"24px\">\n    <g>\n        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n    </g>\n</svg>\n      </button>\n    </div>\n\n<style>\n  .colab-df-quickchart {\n    background-color: #E8F0FE;\n    border: none;\n    border-radius: 50%;\n    cursor: pointer;\n    display: none;\n    fill: #1967D2;\n    height: 32px;\n    padding: 0 0 0 0;\n    width: 32px;\n  }\n\n  .colab-df-quickchart:hover {\n    background-color: #E2EBFA;\n    box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n    fill: #174EA6;\n  }\n\n  [theme=dark] .colab-df-quickchart {\n    background-color: #3B4455;\n    fill: #D2E3FC;\n  }\n\n  [theme=dark] .colab-df-quickchart:hover {\n    background-color: #434B5C;\n    box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n    filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n    fill: #FFFFFF;\n  }\n</style>\n\n    <script>\n      async function quickchart(key) {\n        const containerElement = document.querySelector('#' + key);\n        const charts = await google.colab.kernel.invokeFunction(\n            'suggestCharts', [key], {});\n      }\n    </script>\n`;\n        parentElement.appendChild(quickchartButtonContainerElement);\n        \nfunction displayQuickchartButton(domScope) {\n  let quickchartButtonEl =\n    domScope.querySelector('#df-b4c52e64-21c2-4d5d-bd8a-c20b9852e44b button.colab-df-quickchart');\n  quickchartButtonEl.style.display =\n    google.colab.kernel.accessAllowed ? 'block' : 'none';\n}\n\n        displayQuickchartButton(parentElement);\n      }\n\n      appendQuickchartButton(table);\n    ",
            "text/plain": [
              "<google.colab.data_table.DataTable object>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "9j8fhauSGqYc"
      },
      "id": "9j8fhauSGqYc",
      "execution_count": 6,
      "outputs": []
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "id": "2f136e9c-e208-4cc1-95f7-895266e4adbb",
      "metadata": {
        "tags": [],
        "id": "2f136e9c-e208-4cc1-95f7-895266e4adbb",
        "outputId": "b4b446d7-3008-41ec-d062-d8ed512acba3",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 905
        },
        "cellView": "form"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "49 test requests have been excluded!\n",
            "Number of data access requests : 4\n",
            "Number of unique requestors    : 3\n",
            "Number of unique studies       : 4\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Requestor</th>\n",
              "      <th>Affiliation</th>\n",
              "      <th>Project</th>\n",
              "      <th>Date of approval</th>\n",
              "      <th>Request status</th>\n",
              "      <th>Public Research Use Statement</th>\n",
              "      <th>Technical Research Use Statement</th>\n",
              "      <th>accession</th>\n",
              "      <th>name</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>Miguez, Maria-Jose</td>\n",
              "      <td>FLORIDA INTERNATIONAL UNIVERSITY</td>\n",
              "      <td>System analysis for COVID humoral response</td>\n",
              "      <td>Feb10, 2023</td>\n",
              "      <td>approved</td>\n",
              "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
              "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
              "      <td>phs002945.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>Miguez, Maria-Jose</td>\n",
              "      <td>FLORIDA INTERNATIONAL UNIVERSITY</td>\n",
              "      <td>System analysis for COVID humoral response</td>\n",
              "      <td>Feb10, 2023</td>\n",
              "      <td>approved</td>\n",
              "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
              "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
              "      <td>phs002781.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>47</th>\n",
              "      <td>Solo-Gabriele, Helena</td>\n",
              "      <td>UNIVERSITY OF MIAMI CORAL GABLES</td>\n",
              "      <td>Request to access SF-RAD: Development of Proof...</td>\n",
              "      <td>Jul26, 2023</td>\n",
              "      <td>approved</td>\n",
              "      <td>I am the Principal Investigator for the projec...</td>\n",
              "      <td>I am the Principal Investigator for the projec...</td>\n",
              "      <td>phs002525.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>49</th>\n",
              "      <td>Anwar, Mohd Mozharul</td>\n",
              "      <td>NIH</td>\n",
              "      <td>Exploration of Wearable Device Data for COVID-19</td>\n",
              "      <td>Mar27, 2023</td>\n",
              "      <td>approved</td>\n",
              "      <td>Wearable devices collect various physiological...</td>\n",
              "      <td>The objective of the proposed research is to e...</td>\n",
              "      <td>phs002523.v1.p1</td>\n",
              "      <td>Rapid Acceleration of Diagnostics - Radical (R...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/881c4a0d49046431/data_table.js\";\n\n      const table = window.createDataTable({\n        data: [[\"Miguez, Maria-Jose \",\n\"FLORIDA INTERNATIONAL UNIVERSITY\",\n\"System analysis for COVID humoral response\",\n\"Feb10, 2023\",\n\"approved\",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Therefore, it is critical to understand who is at risk (e.g. those with allergies). Our long term goal is to improve early diagnosis.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely prepared given our medical and research experience.   \",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Indeed, in a sizable proportion of cases (up to 40%), the patient requires to be admitted to the ICU service. Therefore, it is critical to understand who is at risk (e.g. those with allergic or non-allergic diseases) those with alterations in the IgE network (platelets, eosinophils, specific Th2 or Th17 cytokines). Our long term objective is to develop new strategies for early diagnosis and to identify biomarkers of risk that can be easily deployed globally.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely qualified to do such analyses given the medical background of the PI, our ongoing work in COVID both in humans and with animal models, our unique expertise in IgE, and expertise using machine learning and model-based analyses.   \",\n\"phs002945.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Data Science    Approach to Identify and Manage Multisystem Inflammatory Syndrome in    Children (MIS-C) Associated with SARS-CoV-2 Infection and Kawasaki    Disease in Pediatric Patients\"],\n [\"Miguez, Maria-Jose \",\n\"FLORIDA INTERNATIONAL UNIVERSITY\",\n\"System analysis for COVID humoral response\",\n\"Feb10, 2023\",\n\"approved\",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Therefore, it is critical to understand who is at risk (e.g. those with allergies). Our long term goal is to improve early diagnosis.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely prepared given our medical and research experience.   \",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Indeed, in a sizable proportion of cases (up to 40%), the patient requires to be admitted to the ICU service. Therefore, it is critical to understand who is at risk (e.g. those with allergic or non-allergic diseases) those with alterations in the IgE network (platelets, eosinophils, specific Th2 or Th17 cytokines). Our long term objective is to develop new strategies for early diagnosis and to identify biomarkers of risk that can be easily deployed globally.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely qualified to do such analyses given the medical background of the PI, our ongoing work in COVID both in humans and with animal models, our unique expertise in IgE, and expertise using machine learning and model-based analyses.   \",\n\"phs002781.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Discovery and    Clinical Validation of Host Biomarkers of Disease Severity and    Multi-system Inflammatory Syndrome in Children (MIS-C) and COVID-19\"],\n [\"Solo-Gabriele, Helena \",\n\"UNIVERSITY OF MIAMI CORAL GABLES\",\n\"Request to access SF-RAD: Development of Proof-of-Concept\",\n\"Jul26, 2023\",\n\"approved\",\n\"I am the Principal Investigator for the project and I would like to download the data to see how it looks from the user's perspective.  This will help us improve the data upload.\",\n\"I am the Principal Investigator for the project and I would like to download the data to see how it looks from the user's perspective.  This will help us improve the data upload.\",\n\"phs002525.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SF-RAD:    Development and Proof-of-Concept Implementation of the South Florida    Miami RADx-rad SARS-CoV-2 Wastewater-Based Surveillance Infrastructure\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Mar27, 2023\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002523.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multi-Modal    Wireless COVID Monitoring and Infection Alerts for Concentrated    Populations\"]],\n        columns: [[\"string\", \"Requestor\"], [\"string\", \"Affiliation\"], [\"string\", \"Project\"], [\"string\", \"Date of approval\"], [\"string\", \"Request status\"], [\"string\", \"Public Research Use Statement\"], [\"string\", \"Technical Research Use Statement\"], [\"string\", \"accession\"], [\"string\", \"name\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n      \n      function appendQuickchartButton(parentElement) {\n        let quickchartButtonContainerElement = document.createElement('div');\n        quickchartButtonContainerElement.innerHTML = `\n    <div id=\"df-0ce97f59-633c-4939-8545-55240a613f36\">\n      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-0ce97f59-633c-4939-8545-55240a613f36')\"\n              title=\"Suggest charts.\"\n              style=\"display:none;\">\n\n<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n     width=\"24px\">\n    <g>\n        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n    </g>\n</svg>\n      </button>\n    </div>\n\n<style>\n  .colab-df-quickchart {\n    background-color: #E8F0FE;\n    border: none;\n    border-radius: 50%;\n    cursor: pointer;\n    display: none;\n    fill: #1967D2;\n    height: 32px;\n    padding: 0 0 0 0;\n    width: 32px;\n  }\n\n  .colab-df-quickchart:hover {\n    background-color: #E2EBFA;\n    box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n    fill: #174EA6;\n  }\n\n  [theme=dark] .colab-df-quickchart {\n    background-color: #3B4455;\n    fill: #D2E3FC;\n  }\n\n  [theme=dark] .colab-df-quickchart:hover {\n    background-color: #434B5C;\n    box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n    filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n    fill: #FFFFFF;\n  }\n</style>\n\n    <script>\n      async function quickchart(key) {\n        const containerElement = document.querySelector('#' + key);\n        const charts = await google.colab.kernel.invokeFunction(\n            'suggestCharts', [key], {});\n      }\n    </script>\n`;\n        parentElement.appendChild(quickchartButtonContainerElement);\n        \nfunction displayQuickchartButton(domScope) {\n  let quickchartButtonEl =\n    domScope.querySelector('#df-0ce97f59-633c-4939-8545-55240a613f36 button.colab-df-quickchart');\n  quickchartButtonEl.style.display =\n    google.colab.kernel.accessAllowed ? 'block' : 'none';\n}\n\n        displayQuickchartButton(parentElement);\n      }\n\n      appendQuickchartButton(table);\n    ",
            "text/plain": [
              "<google.colab.data_table.DataTable object>"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "#@title Detailed table of data access requests\n",
        "if exclude_tests:\n",
        "    print(f\"{n_excluded} test requests have been excluded!\")\n",
        "\n",
        "print(\"Number of data access requests :\", requests.shape[0])\n",
        "\n",
        "if requests.shape[0] > 0:\n",
        "    print(\"Number of unique requestors    :\", len(requests[\"Requestor\"].unique()))\n",
        "    print(\"Number of unique studies       :\", len(requests[\"accession\"].unique()))\n",
        "    display(data_table.DataTable(requests, include_index=False, num_rows_per_page=10))"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Publications that cite or mention dbGaP accession numbers (source: Europe PMC)\n",
        "studies[\"dbgap\"] = studies[\"accession\"].apply(lambda s: s.split(\".\")[0])\n",
        "# get list of publications from Europe PMC\n",
        "dbgap_pub = pd.read_csv(\"ftp://ftp.ebi.ac.uk/pub/databases/pmc/TextMinedTerms/dbgap.csv\")\n",
        "pubs = studies.merge(dbgap_pub, on=\"dbgap\")\n",
        "pubs.fillna(\"\", inplace=True)\n",
        "print(\"Number of publications:\", pubs.shape[0])\n",
        "display(data_table.DataTable(pubs, include_index=False, num_rows_per_page=10))"
      ],
      "metadata": {
        "cellView": "form",
        "id": "ODVNdnDi0vju",
        "outputId": "ed77e6fc-51b0-4750-c0c7-d198dc6cb413",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 139
        }
      },
      "id": "ODVNdnDi0vju",
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Number of publications: 0\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>accession</th>\n",
              "      <th>name</th>\n",
              "      <th>description</th>\n",
              "      <th>Study Design</th>\n",
              "      <th>Study Consent</th>\n",
              "      <th>dbgap</th>\n",
              "      <th>PMCID</th>\n",
              "      <th>EXTID</th>\n",
              "      <th>SOURCE</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/881c4a0d49046431/data_table.js\";\n\n      const table = window.createDataTable({\n        data: [],\n        columns: [[\"number\", \"accession\"], [\"number\", \"name\"], [\"number\", \"description\"], [\"number\", \"Study Design\"], [\"number\", \"Study Consent\"], [\"number\", \"dbgap\"], [\"number\", \"PMCID\"], [\"number\", \"EXTID\"], [\"number\", \"SOURCE\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n      \n      function appendQuickchartButton(parentElement) {\n        let quickchartButtonContainerElement = document.createElement('div');\n        quickchartButtonContainerElement.innerHTML = `\n    <div id=\"df-b1fe7cdb-ab67-4d50-a38c-b602b4e83b4b\">\n      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-b1fe7cdb-ab67-4d50-a38c-b602b4e83b4b')\"\n              title=\"Suggest charts.\"\n              style=\"display:none;\">\n\n<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n     width=\"24px\">\n    <g>\n        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n    </g>\n</svg>\n      </button>\n    </div>\n\n<style>\n  .colab-df-quickchart {\n    background-color: #E8F0FE;\n    border: none;\n    border-radius: 50%;\n    cursor: pointer;\n    display: none;\n    fill: #1967D2;\n    height: 32px;\n    padding: 0 0 0 0;\n    width: 32px;\n  }\n\n  .colab-df-quickchart:hover {\n    background-color: #E2EBFA;\n    box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n    fill: #174EA6;\n  }\n\n  [theme=dark] .colab-df-quickchart {\n    background-color: #3B4455;\n    fill: #D2E3FC;\n  }\n\n  [theme=dark] .colab-df-quickchart:hover {\n    background-color: #434B5C;\n    box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n    filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n    fill: #FFFFFF;\n  }\n</style>\n\n    <script>\n      async function quickchart(key) {\n        const containerElement = document.querySelector('#' + key);\n        const charts = await google.colab.kernel.invokeFunction(\n            'suggestCharts', [key], {});\n      }\n    </script>\n`;\n        parentElement.appendChild(quickchartButtonContainerElement);\n        \nfunction displayQuickchartButton(domScope) {\n  let quickchartButtonEl =\n    domScope.querySelector('#df-b1fe7cdb-ab67-4d50-a38c-b602b4e83b4b button.colab-df-quickchart');\n  quickchartButtonEl.style.display =\n    google.colab.kernel.accessAllowed ? 'block' : 'none';\n}\n\n        displayQuickchartButton(parentElement);\n      }\n\n      appendQuickchartButton(table);\n    ",
            "text/plain": [
              "<google.colab.data_table.DataTable object>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Publications and preprints that mention  RADx project (source: Europe PMC)\n",
        "url = f\"https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=%22{project}%22%20AND%20(FIRST_PDATE:2020-06-01)%20AND%20sort_date:y&resultType=core&pageSize=1000&format=json&cursorMark=*\"\n",
        "\n",
        "response = urlopen(url)\n",
        "data_json = json.loads(response.read())\n",
        "\n",
        "df = pd.json_normalize(data_json[\"resultList\"][\"result\"])\n",
        "df.fillna(\"\", inplace=True)\n",
        "\n",
        "df.rename(columns={\"fullTextIdList.fullTextId\": \"fullTextId\"}, inplace=True)\n",
        "\n",
        "if df.shape[0] > 0:\n",
        "    df = df[[\"pmid\", \"fullTextId\", \"title\", \"authorString\", \"pubYear\", \"affiliation\", \"citedByCount\"]]\n",
        "\n",
        "    print(f\"Number of publications and preprints containing the keyword: {project}: {df.shape[0]}\")\n",
        "    print(f\"Number of citations: {df['citedByCount'].sum()}\")\n",
        "    display(data_table.DataTable(df, include_index=False, num_rows_per_page=10))\n",
        "else:\n",
        "    print(f\"Number of publications and preprints containing the keyword: {project}: 0\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 794
        },
        "cellView": "form",
        "id": "dCoRRWMVkdUH",
        "outputId": "f4c7fee2-1d58-478f-a717-8972dda8e738"
      },
      "id": "dCoRRWMVkdUH",
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Number of publications and preprints containing the keyword: RADx-rad: 28\n",
            "Number of citations: 264\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>fullTextId</th>\n",
              "      <th>title</th>\n",
              "      <th>authorString</th>\n",
              "      <th>pubYear</th>\n",
              "      <th>affiliation</th>\n",
              "      <th>citedByCount</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>37429842</td>\n",
              "      <td>[PMC10333287]</td>\n",
              "      <td>Real-time environmental surveillance of SARS-C...</td>\n",
              "      <td>Puthussery JV, Ghumra DP, McBrearty KR, Dohert...</td>\n",
              "      <td>2023</td>\n",
              "      <td>Center for Aerosol Science and Engineering, De...</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td></td>\n",
              "      <td>[PPR668764]</td>\n",
              "      <td>Geospatially-resolved public-health surveillan...</td>\n",
              "      <td>Tierney BT, Foox J, Ryon KA, Butler D, Damle N...</td>\n",
              "      <td>2023</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>37208471</td>\n",
              "      <td>[PMC10199082]</td>\n",
              "      <td>iOBPdb A Database for Experimentally Determine...</td>\n",
              "      <td>Shukla S, Nakano-Baker O, Godin D, MacKenzie D...</td>\n",
              "      <td>2023</td>\n",
              "      <td>University of Washington, Seattle, WA, USA. ss...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>36965380</td>\n",
              "      <td>[PMC10027305]</td>\n",
              "      <td>Data-driven design of a multiplexed, peptide-s...</td>\n",
              "      <td>Nakano-Baker O, Fong H, Shukla S, Lee RV, Cai ...</td>\n",
              "      <td>2023</td>\n",
              "      <td>University of Washington Dept. of Materials Sc...</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>36933724</td>\n",
              "      <td>[PMC10017378]</td>\n",
              "      <td>Wastewater surveillance uncovers regional dive...</td>\n",
              "      <td>Fontenele RS, Yang Y, Driver EM, Magge A, Krab...</td>\n",
              "      <td>2023</td>\n",
              "      <td>National Library of Medicine, National Institu...</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>36623667</td>\n",
              "      <td>[PMC9817413]</td>\n",
              "      <td>Degradation rates influence the ability of com...</td>\n",
              "      <td>Babler KM, Sharkey ME, Abelson S, Amirali A, B...</td>\n",
              "      <td>2023</td>\n",
              "      <td>Department of Chemical, Environmental, and Mat...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>36482179</td>\n",
              "      <td>[PMC9731983]</td>\n",
              "      <td>Predominant SARS-CoV-2 variant impacts accurac...</td>\n",
              "      <td>McCartney MM, Borras E, Rojas DE, Hicks TL, Ha...</td>\n",
              "      <td>2022</td>\n",
              "      <td>Mechanical and Aerospace Engineering, UC Davis...</td>\n",
              "      <td>2</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>36493788</td>\n",
              "      <td>[PMC9725778]</td>\n",
              "      <td>Leveraging an established neighbourhood-level,...</td>\n",
              "      <td>Bowes DA, Driver EM, Kraberger S, Fontenele RS...</td>\n",
              "      <td>2023</td>\n",
              "      <td>The Biodesign Institute Center for Environment...</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>36354449</td>\n",
              "      <td>[PMC9688365]</td>\n",
              "      <td>An Experimental Framework for Developing Point...</td>\n",
              "      <td>Ullah SF, Moreira G, Datta SPA, McLamore E, Va...</td>\n",
              "      <td>2022</td>\n",
              "      <td>Division of Glycoscience, Department of Chemis...</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td></td>\n",
              "      <td>[PPR541952]</td>\n",
              "      <td>Portable Breath-Based Volatile Organic Compoun...</td>\n",
              "      <td>Sharma R, Zang W, Tabartehfarahani A, Lam A, H...</td>\n",
              "      <td>2022</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td></td>\n",
              "      <td>[PPR535354]</td>\n",
              "      <td>Data-Driven Design of Protein-Derived Peptide ...</td>\n",
              "      <td>Nakano-Baker O, Fong H, Shukla S, Lee R, Cai L...</td>\n",
              "      <td>2022</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11</th>\n",
              "      <td>35992634</td>\n",
              "      <td>[PMC9386735]</td>\n",
              "      <td>Development of a Biosensor Based on Angiotensi...</td>\n",
              "      <td>Moreira G, Casso-Hartmann L, Datta SPA, Dean D...</td>\n",
              "      <td>2022</td>\n",
              "      <td>Department of Environmental Engineering and Ea...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td></td>\n",
              "      <td>[PPR511405]</td>\n",
              "      <td>Outbreak.info Research Library: A standardized...</td>\n",
              "      <td>Hughes L, Tsueng G, Mullen J, Alkuzweny M, Can...</td>\n",
              "      <td>2022</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>35632824</td>\n",
              "      <td>[PMC9148162]</td>\n",
              "      <td>Pivoting Novel Exosome-Based Technologies for ...</td>\n",
              "      <td>Happel C, Peñalber-Johnstone C, Tagle DA.</td>\n",
              "      <td>2022</td>\n",
              "      <td>National Center for Advancing Translational Sc...</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>14</th>\n",
              "      <td>35093995</td>\n",
              "      <td>[PMC8919949]</td>\n",
              "      <td>Distinguishing Multisystem Inflammatory Syndro...</td>\n",
              "      <td>Godfred-Cato S, Abrams JY, Balachandran N, Jag...</td>\n",
              "      <td>2022</td>\n",
              "      <td>CDC COVID-19 Response Team, Atlanta, GA.</td>\n",
              "      <td>22</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>15</th>\n",
              "      <td></td>\n",
              "      <td>[PPR471521]</td>\n",
              "      <td>Proof-of-concept: SCENTinel 1.1 rapidly discri...</td>\n",
              "      <td>Hunter SR, Hannum ME, Pellegrino R, O’Leary MA...</td>\n",
              "      <td>2022</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>16</th>\n",
              "      <td></td>\n",
              "      <td>[PPR445454]</td>\n",
              "      <td>Outbreak.info Research Library: A standardized...</td>\n",
              "      <td>Tsueng G, Mullen JL, Alkuzweny M, Cano M, Rush...</td>\n",
              "      <td>2022</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>17</th>\n",
              "      <td>34893640</td>\n",
              "      <td>[PMC8664884]</td>\n",
              "      <td>Proteomic profiling of MIS-C patients indicate...</td>\n",
              "      <td>Diorio C, Shraim R, Vella LA, Giles JR, Baxter...</td>\n",
              "      <td>2021</td>\n",
              "      <td>Division of Oncology, Department of Pediatrics...</td>\n",
              "      <td>18</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>18</th>\n",
              "      <td>34778896</td>\n",
              "      <td>[PMC8860149]</td>\n",
              "      <td>Point-of-care diagnostics: recent developments...</td>\n",
              "      <td>Harpaldas H, Arumugam S, Campillo Rodriguez C,...</td>\n",
              "      <td>2021</td>\n",
              "      <td>Department of Biomedical Engineering, Columbia...</td>\n",
              "      <td>8</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>19</th>\n",
              "      <td></td>\n",
              "      <td>[PPR377134]</td>\n",
              "      <td>Unrestricted Online Sharing of High-frequency,...</td>\n",
              "      <td>Bowes DA, Driver EM, Kraberger S, Fontenele RS...</td>\n",
              "      <td>2021</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>20</th>\n",
              "      <td>34611607</td>\n",
              "      <td>[PMC8486427]</td>\n",
              "      <td>Biomedical Research COVID-19 Impact Assessment...</td>\n",
              "      <td>Cook NL, Lauer MS.</td>\n",
              "      <td>2021</td>\n",
              "      <td>Patient-Centered Outcomes Research Institute.</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>21</th>\n",
              "      <td>34514084</td>\n",
              "      <td>[PMC8420437]</td>\n",
              "      <td>SARS-CoV-2 RNA Detection by a Cellphone-Based ...</td>\n",
              "      <td>Silva FSR, Erdogmus E, Shokr A, Kandula H, Thi...</td>\n",
              "      <td>2021</td>\n",
              "      <td>Division of Engineering in Medicine Department...</td>\n",
              "      <td>18</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>22</th>\n",
              "      <td></td>\n",
              "      <td>[PPR314466]</td>\n",
              "      <td>Proteomic Profiling of MIS-C Patients Reveals ...</td>\n",
              "      <td>Diorio C, Shraim R, Vella LA, Giles JR, Baxter...</td>\n",
              "      <td>2021</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>23</th>\n",
              "      <td>33824513</td>\n",
              "      <td>[PMC8023346]</td>\n",
              "      <td>Radical solutions.</td>\n",
              "      <td></td>\n",
              "      <td>2021</td>\n",
              "      <td></td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>24</th>\n",
              "      <td>33773496</td>\n",
              "      <td>[PMC8083606]</td>\n",
              "      <td>SCENTinel 1.0: Development of a Rapid Test to ...</td>\n",
              "      <td>Parma V, Hannum ME, O'Leary M, Pellegrino R, R...</td>\n",
              "      <td>2021</td>\n",
              "      <td>Department of Psychology, Temple University, 1...</td>\n",
              "      <td>13</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>25</th>\n",
              "      <td></td>\n",
              "      <td>[PPR251997]</td>\n",
              "      <td>&lt;i&gt;SCENTinel 1.0&lt;/i&gt;: development of a rapid t...</td>\n",
              "      <td>Parma V, Hannum ME, O’Leary M, Pellegrino R, R...</td>\n",
              "      <td>2020</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>26</th>\n",
              "      <td>33073312</td>\n",
              "      <td>[PMC7568947]</td>\n",
              "      <td>SARS-CoV-2 pandemic: a review of molecular dia...</td>\n",
              "      <td>Jayamohan H, Lambert CJ, Sant HJ, Jafek A, Pat...</td>\n",
              "      <td>2021</td>\n",
              "      <td>Department of Mechanical Engineering, Universi...</td>\n",
              "      <td>66</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>27</th>\n",
              "      <td>32706958</td>\n",
              "      <td>[PMC7493127]</td>\n",
              "      <td>Rapid Scaling Up of Covid-19 Diagnostic Testin...</td>\n",
              "      <td>Tromberg BJ, Schwetz TA, Pérez-Stable EJ, Hode...</td>\n",
              "      <td>2020</td>\n",
              "      <td>From the National Institute of Biomedical Imag...</td>\n",
              "      <td>109</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/881c4a0d49046431/data_table.js\";\n\n      const table = window.createDataTable({\n        data: [[\"37429842\",\n[\"PMC10333287\"],\n\"Real-time environmental surveillance of SARS-CoV-2 aerosols.\",\n\"Puthussery JV, Ghumra DP, McBrearty KR, Doherty BM, Sumlin BJ, Sarabandi A, Mandal AG, Shetty NJ, Gardiner WD, Magrecki JP, Brody DL, Esparza TJ, Bricker TL, Boon ACM, Yuede CM, Cirrito JR, Chakrabarty RK.\",\n\"2023\",\n\"Center for Aerosol Science and Engineering, Department of Energy, Environmental and Chemical Engineering, Washington University in St. Louis, St. Louis, MO, 63130, USA.\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"\",\n[\"PPR668764\"],\n\"Geospatially-resolved public-health surveillance via wastewater sequencing\",\n\"Tierney BT, Foox J, Ryon KA, Butler D, Damle N, Young BG, Mozsary C, Babler KM, Yin X, Carattini Y, Andrews D, Solle NS, Kumar N, Shukla B, Vidovic D, Currall B, Williams SL, Sch\\u00fcrer SC, Stevenson M, Amirali A, Beaver CC, Kobetz E, Boone MM, Reding B, Laine J, Comerford S, Lamar WE, Tallon JJ, Hirschberg JW, Proszynski J, Sharkey ME, Church GM, Grills GS, Solo-Gabriele HM, Mason CE.\",\n\"2023\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"37208471\",\n[\"PMC10199082\"],\n\"iOBPdb A Database for Experimentally Determined Functional Characterization of Insect Odorant Binding Proteins.\",\n\"Shukla S, Nakano-Baker O, Godin D, MacKenzie D, Sarikaya M.\",\n\"2023\",\n\"University of Washington, Seattle, WA, USA. sshuklz4@uw.edu.\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"36965380\",\n[\"PMC10027305\"],\n\"Data-driven design of a multiplexed, peptide-sensitized transistor to detect breath VOC markers of COVID-19.\",\n\"Nakano-Baker O, Fong H, Shukla S, Lee RV, Cai L, Godin D, Hennig T, Rath S, Novosselov I, Dogan S, Sarikaya M, MacKenzie JD.\",\n\"2023\",\n\"University of Washington Dept. of Materials Science and Engineering, USA. Electronic address: onb@uw.edu.\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"36933724\",\n[\"PMC10017378\"],\n\"Wastewater surveillance uncovers regional diversity and dynamics of SARS-CoV-2 variants across nine states in the USA.\",\n\"Fontenele RS, Yang Y, Driver EM, Magge A, Kraberger S, Custer JM, Dufault-Thompson K, Cox E, Newell ME, Varsani A, Halden RU, Scotch M, Jiang X.\",\n\"2023\",\n\"National Library of Medicine, National Institute of Health, 8600 Rockville Pike, Bethesda, MD 20894, USA.\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"36623667\",\n[\"PMC9817413\"],\n\"Degradation rates influence the ability of composite samples to represent 24-hourly means of SARS-CoV-2 and other microbiological target measures in wastewater.\",\n\"Babler KM, Sharkey ME, Abelson S, Amirali A, Benitez A, Cosculluela GA, Grills GS, Kumar N, Laine J, Lamar W, Lamm ED, Lyu J, Mason CE, McCabe PM, Raghavender J, Reding BD, Roca MA, Sch\\u00fcrer SC, Stevenson M, Szeto A, Tallon JJ, Vidovi\\u0107 D, Zarnegarnia Y, Solo-Gabriele HM.\",\n\"2023\",\n\"Department of Chemical, Environmental, and Materials Engineering, University of Miami, Coral Gables, FL 33146, USA.\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"36482179\",\n[\"PMC9731983\"],\n\"Predominant SARS-CoV-2 variant impacts accuracy when screening for infection using exhaled breath vapor.\",\n\"McCartney MM, Borras E, Rojas DE, Hicks TL, Hamera KL, Tran NK, Tham T, Juarez MM, Lopez E, Kenyon NJ, Davis CE.\",\n\"2022\",\n\"Mechanical and Aerospace Engineering, UC Davis, Davis, CA, USA.\",\n{\n            'v': 2,\n            'f': \"2\",\n        }],\n [\"36493788\",\n[\"PMC9725778\"],\n\"Leveraging an established neighbourhood-level, open access wastewater monitoring network to address public health priorities: a population-based study.\",\n\"Bowes DA, Driver EM, Kraberger S, Fontenele RS, Holland LA, Wright J, Johnston B, Savic S, Engstrom Newell M, Adhikari S, Kumar R, Goetz H, Binsfeld A, Nessi K, Watkins P, Mahant A, Zevitz J, Deitrick S, Brown P, Dalton R, Garcia C, Inchausti R, Holmes W, Tian XJ, Varsani A, Lim ES, Scotch M, Halden RU.\",\n\"2023\",\n\"The Biodesign Institute Center for Environmental Health Engineering, Arizona State University, Tempe, AZ, USA; School for Engineering of Matter, Transport, and Energy, Arizona State University, Tempe, AZ, USA; OneWaterOneHealth, The Arizona State University Foundation, The Biodesign Institute, Arizona State University, Tempe, AZ, USA.\",\n{\n            'v': 3,\n            'f': \"3\",\n        }],\n [\"36354449\",\n[\"PMC9688365\"],\n\"An Experimental Framework for Developing Point-of-Need Biosensors: Connecting Bio-Layer Interferometry and Electrochemical Impedance Spectroscopy.\",\n\"Ullah SF, Moreira G, Datta SPA, McLamore E, Vanegas D.\",\n\"2022\",\n\"Division of Glycoscience, Department of Chemistry, School of Engineering Sciences in Chemistry, Biotechnology and Health, KTH Royal Institute of Technology, 106 91 Stockholm, Sweden.\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"\",\n[\"PPR541952\"],\n\"Portable Breath-Based Volatile Organic Compound Monitoring for the Detection of COVID-19: Challenges of Emerging Variants\",\n\"Sharma R, Zang W, Tabartehfarahani A, Lam A, Huang X, Sivakumar AD, Thota C, Yang S, Dickson RP, Sjoding MW, Bisco E, Mahmood CC, Diaz KM, Sautter N, Ansari S, Ward KR, Fan X.\",\n\"2022\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"\",\n[\"PPR535354\"],\n\"Data-Driven Design of Protein-Derived Peptide Multiplexes for Biomimetic Detection of Exhaled Breath VOC Profiles\",\n\"Nakano-Baker O, Fong H, Shukla S, Lee R, Cai L, Godin D, Hennig T, Rath S, Novosselov I, Dogan S, Sarikaya M, MacKenzie JD.\",\n\"2022\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"35992634\",\n[\"PMC9386735\"],\n\"Development of a Biosensor Based on Angiotensin-Converting Enzyme II for Severe Acute Respiratory Syndrome Coronavirus 2 Detection in Human Saliva.\",\n\"Moreira G, Casso-Hartmann L, Datta SPA, Dean D, McLamore E, Vanegas D.\",\n\"2022\",\n\"Department of Environmental Engineering and Earth Sciences, Clemson University, Clemson, SC, United States.\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"\",\n[\"PPR511405\"],\n\"Outbreak.info Research Library: A standardized, searchable platform to discover and explore COVID-19 resources\",\n\"Hughes L, Tsueng G, Mullen J, Alkuzweny M, Cano M, Rush B, Haag E, Latif AA, Zhou X, Qian Z, Hufbauer E, Zeller M, Andersen K, Wu C, Su A, Gangavarapu K.\",\n\"2022\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"35632824\",\n[\"PMC9148162\"],\n\"Pivoting Novel Exosome-Based Technologies for the Detection of SARS-CoV-2.\",\n\"Happel C, Pe\\u00f1alber-Johnstone C, Tagle DA.\",\n\"2022\",\n\"National Center for Advancing Translational Sciences, National Institutes of Health, Bethesda, MD 20892, USA.\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"35093995\",\n[\"PMC8919949\"],\n\"Distinguishing Multisystem Inflammatory Syndrome in Children From COVID-19, Kawasaki Disease and Toxic Shock Syndrome.\",\n\"Godfred-Cato S, Abrams JY, Balachandran N, Jaggi P, Jones K, Rostad CA, Lu AT, Fan L, Jabbar A, Anderson EJ, Kao CM, Hunstad DA, Rosenberg RB, Zafferani MJ, Ede KC, Ballan W, Laham FR, Beltran Y, Bryant B, Meng L, Hammett TA, Oster ME, Bamrah Morris S, Belay ED.\",\n\"2022\",\n\"CDC COVID-19 Response Team, Atlanta, GA.\",\n{\n            'v': 22,\n            'f': \"22\",\n        }],\n [\"\",\n[\"PPR471521\"],\n\"Proof-of-concept: SCENTinel 1.1 rapidly discriminates COVID-19 related olfactory disorders\",\n\"Hunter SR, Hannum ME, Pellegrino R, O\\u2019Leary MA, Rawson NE, Reed DR, Dalton PH, Parma V.\",\n\"2022\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"\",\n[\"PPR445454\"],\n\"Outbreak.info Research Library: A standardized, searchable platform to discover and explore COVID-19 resources\",\n\"Tsueng G, Mullen JL, Alkuzweny M, Cano M, Rush B, Haag E, Lin J, Welzel DJ, Zhou X, Qian Z, Latif AA, Hufbauer E, Zeller M, Andersen KG, Wu C, Su AI, Gangavarapu K, Hughes LD, Outbreak Curators.\",\n\"2022\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"34893640\",\n[\"PMC8664884\"],\n\"Proteomic profiling of MIS-C patients indicates heterogeneity relating to interferon gamma dysregulation and vascular endothelial dysfunction.\",\n\"Diorio C, Shraim R, Vella LA, Giles JR, Baxter AE, Oldridge DA, Canna SW, Henrickson SE, McNerney KO, Balamuth F, Burudpakdee C, Lee J, Leng T, Farrel A, Lambert MP, Sullivan KE, Wherry EJ, Teachey DT, Bassiri H, Behrens EM.\",\n\"2021\",\n\"Division of Oncology, Department of Pediatrics, Children's Hospital of Philadelphia, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA, USA.\",\n{\n            'v': 18,\n            'f': \"18\",\n        }],\n [\"34778896\",\n[\"PMC8860149\"],\n\"Point-of-care diagnostics: recent developments in a pandemic age.\",\n\"Harpaldas H, Arumugam S, Campillo Rodriguez C, Kumar BA, Shi V, Sia SK.\",\n\"2021\",\n\"Department of Biomedical Engineering, Columbia University, New York, NY 10027, USA. ss2735@columbia.edu.\",\n{\n            'v': 8,\n            'f': \"8\",\n        }],\n [\"\",\n[\"PPR377134\"],\n\"Unrestricted Online Sharing of High-frequency, High-resolution Data on SARS-CoV-2 in Wastewater to Inform the COVID-19 Public Health Response in Greater Tempe, Arizona\",\n\"Bowes DA, Driver EM, Kraberger S, Fontenele RS, Holland LA, Wright J, Johnston B, Savic S, Newell ME, Adhikari S, Kumar R, Goetz H, Binsfeld A, Nessi K, Watkins P, Mahant A, Zevitz J, Deitrick S, Brown P, Dalton R, Garcia C, Inchausti R, Holmes W, Tian X, Varsani A, Lim ES, Scotch M, Halden RU.\",\n\"2021\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"34611607\",\n[\"PMC8486427\"],\n\"Biomedical Research COVID-19 Impact Assessment: Lessons Learned and Compelling Needs. \",\n\"Cook NL, Lauer MS.\",\n\"2021\",\n\"Patient-Centered Outcomes Research Institute.\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"34514084\",\n[\"PMC8420437\"],\n\"SARS-CoV-2 RNA Detection by a Cellphone-Based Amplification-Free System with CRISPR/CAS-Dependent Enzymatic (CASCADE) Assay.\",\n\"Silva FSR, Erdogmus E, Shokr A, Kandula H, Thirumalaraju P, Kanakasabapathy MK, Hardie JM, Pacheco LGC, Li JZ, Kuritzkes DR, Shafiee H.\",\n\"2021\",\n\"Division of Engineering in Medicine Department of Medicine Brigham and Women's Hospital Harvard Medical School Boston MA 02139 USA.\",\n{\n            'v': 18,\n            'f': \"18\",\n        }],\n [\"\",\n[\"PPR314466\"],\n\"Proteomic Profiling of MIS-C Patients Reveals Heterogeneity Relating to Interferon Gamma Dysregulation and Vascular Endothelial Dysfunction\",\n\"Diorio C, Shraim R, Vella LA, Giles JR, Baxter AE, Oldridge DA, Canna SW, Henrickson SE, McNerney KO, Balamuth F, Burudpakdee C, Lee J, Leng T, Farrell A, Lambert MP, Sullivan KE, John Wherry E, Teachey DT, Bassiri H, Behrens EM.\",\n\"2021\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"33824513\",\n[\"PMC8023346\"],\n\"Radical solutions.\",\n\"\",\n\"2021\",\n\"\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"33773496\",\n[\"PMC8083606\"],\n\"SCENTinel 1.0: Development of a Rapid Test to Screen for Smell Loss. \",\n\"Parma V, Hannum ME, O'Leary M, Pellegrino R, Rawson NE, Reed DR, Dalton PH.\",\n\"2021\",\n\"Department of Psychology, Temple University, 1701 N 13th Street, Philadelphia, PA, USA.\",\n{\n            'v': 13,\n            'f': \"13\",\n        }],\n [\"\",\n[\"PPR251997\"],\n\"<i>SCENTinel 1.0<\\/i>: development of a rapid test to screen for smell loss\",\n\"Parma V, Hannum ME, O\\u2019Leary M, Pellegrino R, Rawson NE, Reed DR, Dalton PH.\",\n\"2020\",\n\"\",\n{\n            'v': 0,\n            'f': \"0\",\n        }],\n [\"33073312\",\n[\"PMC7568947\"],\n\"SARS-CoV-2 pandemic: a review of molecular diagnostic tools including sample collection and commercial response with associated advantages and limitations.\",\n\"Jayamohan H, Lambert CJ, Sant HJ, Jafek A, Patel D, Feng H, Beeman M, Mahmood T, Nze U, Gale BK.\",\n\"2021\",\n\"Department of Mechanical Engineering, University of Utah, Salt Lake City, UT, 84112, USA. hari.jayamohan@utah.edu.\",\n{\n            'v': 66,\n            'f': \"66\",\n        }],\n [\"32706958\",\n[\"PMC7493127\"],\n\"Rapid Scaling Up of Covid-19 Diagnostic Testing in the United States - The NIH RADx Initiative.\",\n\"Tromberg BJ, Schwetz TA, P\\u00e9rez-Stable EJ, Hodes RJ, Woychik RP, Bright RA, Fleurence RL, Collins FS.\",\n\"2020\",\n\"From the National Institute of Biomedical Imaging and Bioengineering (B.J.T.), the National Institute of Nursing Research (T.A.S.), the Office of the Director (T.A.S., R.A.B., R.L.F., F.S.C.), the National Institute on Minority Health and Health Disparities (E.J.P.-S.), the National Institute on Aging (R.J.H.), and the National Institute of Environmental Health Sciences, National Toxicology Program (R.P.W.), National Institutes of Health, Bethesda, MD.\",\n{\n            'v': 109,\n            'f': \"109\",\n        }]],\n        columns: [[\"string\", \"pmid\"], [\"string\", \"fullTextId\"], [\"string\", \"title\"], [\"string\", \"authorString\"], [\"string\", \"pubYear\"], [\"string\", \"affiliation\"], [\"number\", \"citedByCount\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n      \n      function appendQuickchartButton(parentElement) {\n        let quickchartButtonContainerElement = document.createElement('div');\n        quickchartButtonContainerElement.innerHTML = `\n    <div id=\"df-902530e2-a83b-4ad4-a184-60c1f38eb592\">\n      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-902530e2-a83b-4ad4-a184-60c1f38eb592')\"\n              title=\"Suggest charts.\"\n              style=\"display:none;\">\n\n<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n     width=\"24px\">\n    <g>\n        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n    </g>\n</svg>\n      </button>\n    </div>\n\n<style>\n  .colab-df-quickchart {\n    background-color: #E8F0FE;\n    border: none;\n    border-radius: 50%;\n    cursor: pointer;\n    display: none;\n    fill: #1967D2;\n    height: 32px;\n    padding: 0 0 0 0;\n    width: 32px;\n  }\n\n  .colab-df-quickchart:hover {\n    background-color: #E2EBFA;\n    box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n    fill: #174EA6;\n  }\n\n  [theme=dark] .colab-df-quickchart {\n    background-color: #3B4455;\n    fill: #D2E3FC;\n  }\n\n  [theme=dark] .colab-df-quickchart:hover {\n    background-color: #434B5C;\n    box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n    filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n    fill: #FFFFFF;\n  }\n</style>\n\n    <script>\n      async function quickchart(key) {\n        const containerElement = document.querySelector('#' + key);\n        const charts = await google.colab.kernel.invokeFunction(\n            'suggestCharts', [key], {});\n      }\n    </script>\n`;\n        parentElement.appendChild(quickchartButtonContainerElement);\n        \nfunction displayQuickchartButton(domScope) {\n  let quickchartButtonEl =\n    domScope.querySelector('#df-902530e2-a83b-4ad4-a184-60c1f38eb592 button.colab-df-quickchart');\n  quickchartButtonEl.style.display =\n    google.colab.kernel.accessAllowed ? 'block' : 'none';\n}\n\n        displayQuickchartButton(parentElement);\n      }\n\n      appendQuickchartButton(table);\n    ",
            "text/plain": [
              "<google.colab.data_table.DataTable object>"
            ]
          },
          "metadata": {}
        }
      ]
    }
  ],
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3 (ipykernel)",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.8.16"
    },
    "colab": {
      "provenance": []
    }
  },
  "nbformat": 4,
  "nbformat_minor": 5
}